src/main/java/stream/flarebot/flarebot/commands/music/SkipCommand.java

```diff
@@ -163,8 +163,9 @@ public String getDescription() {
 
     @Override
     public String getUsage() {
-        return "{%}skip [yes/no]\n" +
-                "{%}skip force";
+        return "`{%}skip` - Starts a vote to skip the song\n" +
+                "`{%}skip yes|no` - Vote yes or no to skip the current song\n" +
```

Comment:
We should tell them that this is only used when a vote is running and to start a vote run `_skip`

Reply:
Read the first line, it tells them how to start a vote. The error messages in the commands explain the rest

Reply:
Fair enough